plugins/zfs/zpool_fragmentation

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#%# family=auto
+#%# capabilities=autoconf
+
+# Michael Grote
+# michael.grote ät posteo.de
+# Outputs the zpool fragmentation per zfs pool.
+
+# lese alle pools, sed löscht die erste zeile
+# entferne das %-zeichen
+list=$(zpool list | sed 1d | tr -d %)
+
+# wenn parameter = ...
+if [ "$1" = "autoconf" ]; then
```

Comment:
Please verify the availability of `zpool`.

Reply:
added